pkg/utils/apiutil/apiutil.go

```diff
@@ -477,3 +478,15 @@ func copyHeader(dst, src http.Header) {
 		}
 	}
 }
+
+// ParseTime parses a time string with the format "1694580288"
+func ParseTime(t string) (time.Time, error) {
+	if len(t) == 0 {
+		return time.Time{}, nil
```

Comment:
Do we need to return an error?

Reply:
No, for http api if there is no "from", it will return all results.